### ISSUE TITLE
SuperSpy Glass

### ISSUE THREAD

**Issue by SharpShotGif** [cadon/ARKStatsExtractor#645]
This is more of a suggestion than a problem. 

Any chance of you making it so it can extract the number from the SSG as an alt method of adding to the Library. The SSG shows the wild and dom levels.

http://steamcommunity.com/sharedfiles/filedetails/?id=793605978

**Comment by SharpShotGif** [comment]
The new TLC patch added a export dino option. This exports the dinos stats in a ini file. This could make importing data 100% accurate meaning the OCR might no longer be needed.

**Comment by defyreality** [comment]
for PC. still helps Console players

**Comment by cadon** [comment]
Consoles have no way to install the super spy glass mod, though, or was this possibility added?

**Comment by defyreality** [comment]
no, on console we can't install any of the awesome mods, so OCR is still very much useful for us

**Comment by cadon** [comment]
Just to be clear, I won't remove the OCR. I think SharpShot meant that the additional OCR functionality he requested in this issue for reading the spyglass-values isn't necessary anymore. The existing OCR for reading the inventory values is not affected by that.
